File: Ohtap E Commerce Domain Models/Development/PHP/Objects/PackageAttributes/IPackageAttributeData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\PackageAttributes;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IPackageAttributeData extends IPackageAttributeAttribute,IPackageAttributeValue,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/PackageAttributes/IPackageAttributeData.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.PackageAttributes
{
    public interface IPackageAttributeData :  CoreObject.IObjectID, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IPackageAttributeValue, IPackageAttributeAttribute
    {

    }
}
